workorders.lic

```diff
@@ -715,7 +717,9 @@ class WorkOrders
         end
         fput('get logbook') unless [left_hand, right_hand].grep(/logbook/i).any?
       end
+      $i +=1
     end
+    exit
```

Comment:
careful with this. see https://github.com/rpherbig/dr-scripts/issues/952

Reply:
This one should be ok since no crafting script is running. This loop is the sole responsibility of `workorders`, so it can exit. Make sure there's no logbook in hand, though.